attributes/default.rb

```diff
@@ -1,10 +1,12 @@
+default['mailcatcher']['ruby_version'] = '1.9.3-p545'
+
 default['mailcatcher']['http_ip'] = '0.0.0.0'
 default['mailcatcher']['http_port'] = 1080
 
 default['mailcatcher']['smtp_ip'] = '127.0.0.1'
 default['mailcatcher']['smtp_port'] = 1025
 
-default['mailcatcher']['bin'] = 'mailcatcher'
+default['mailcatcher']['bin'] = '/opt/rbenv/shims/mailcatcher'
```

Comment:
I will try again to see if I assumed something wrong, but the first time I tried it, whenever I tried `sudo /etc/init.d/mailcacther start/restart` it was complaining that there is no `mailcatcher` command (i.e. it was in the shims, but somehow only available to `rbenv` user which was created by rbenv recipe).
It might be that all I actually needed was to restart my shell session, so I will get back on this.
Eitherway, I don't think there would be path problems, because we are installing the rbenv, so it should be safe to assume the standard paths that the recipe uses (maybe worth a check if that's configurable in the recipe - in which case we could reuse the path prefix I suppose)